src/test/java/plugins/ConfigFileProviderTest.java

```diff
@@ -129,4 +139,133 @@ private String createPipelineAndGetConsole(final CustomConfig customConfig) {
         final Build b = job.startBuild().shouldSucceed();
         return b.getConsole();
     }
+
+    @Test
+    public void testConfigFileProviderWithFolders() {
+        final MavenSettingsConfig mvnConfig = this.createMavenSettingsConfig(SERVER_ID, ANOTHER_SERVER_ID, CRED_ID);
+
+        final Folder f = jenkins.jobs.create(Folder.class);
+        final CustomConfig customConf = this.createCustomConfigInFolder(f, CUSTOM_CONF_CONTENT);
+        final WorkflowJob job = createPipelineJobInFolderWithScript(f, scriptForPipelineWithParameters(mvnConfig.id(), customConf.id()));
+
+        String jobLog = this.buildJobAndGetConsole(job, true);
+
+        Assert.assertThat(jobLog, containsString(SERVER_ID));
```

Comment:
ATH uses hamcrest for #assertThat, not JUnit.

Reply:
good point. fixed in 368608e0